File: abc/HelloWorld/Lambda.php
<?php
    /*
     *  Just for Fun
     *
     *  Author:       vac
     *  Organization: STEM
     *  LICENCE:      MIT
     *  Copyright:    2014-2018
     */

    // function declaration
    function sayHello(): void {
        // output some text
        print("hello, world\n");
    }

    // entry point of the program
    if (count(debug_backtrace()) == 0) {
        // function invocation
        sayHello();
    }
?>

File: abc/HelloWorld/Lambda.hs
---
--- Just for Fun
---
--- Author:       vac
--- Organization: STEM
--- LICENCE:      MIT
--- Copyright:    2014-2018

module Main (main) where

-- function declaration
sayHello :: IO ()
sayHello =
    putStrLn "hello, world" -- output some text

-- entry point of the program
main :: IO ()
main =
    sayHello -- function invocation
